src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -27,16 +30,81 @@ public AuthenticationTests()
 
         public void Dispose() => _factory.Dispose();
 
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // Managed implementation connects successfully.
+        [InlineData("Basic")]
+        [InlineData("NTLM")]
+        [InlineData("Negotiate")]
+        [InlineData("Unknown")]
+        public async Task NoAuthentication_AuthenticationProvided_ReturnsForbiddenStatusCode(string headerType)
+        {
+            _listener.AuthenticationSchemes = AuthenticationSchemes.None;
+
+            using (HttpClient client = new HttpClient())
+            {
+                client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue(headerType, "body");
+                await AuthenticationFailure(client, HttpStatusCode.Forbidden);
+            }
+        }
+
         [ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
         [InlineData(AuthenticationSchemes.Basic)]
         [InlineData(AuthenticationSchemes.Basic | AuthenticationSchemes.None)]
         [InlineData(AuthenticationSchemes.Basic | AuthenticationSchemes.Anonymous)]
-        public async Task TestBasicAuthentication(AuthenticationSchemes authScheme)
+        public async Task BasicAuthentication_ValidUsernameAndPassword_Success(AuthenticationSchemes authScheme)
         {
             _listener.AuthenticationSchemes = authScheme;
             await ValidateValidUser();
         }
 
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // [ActiveIssue(20099, TestPlatforms.Unix)]
```

Comment:
Why is this ConditionalTheory construct being used like this with IsWindowsImplementation?  It doesn't work to just add the active issue for unix?
```C#
[ActiveIssue(20099, TestPlatforms.Unix)]
[ConditionalTheory(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotOneCoreUAP))]
```
?